src/components/SwapWidget/LimitOrder/index.tsx

```diff
@@ -506,7 +511,7 @@ const LimitOrder: React.FC<Props> = ({ swapType, setSwapType, isLimitOrderVisibl
               setTokenDrawerType(LimitNewField.OUTPUT as any);
               setIsTokenDrawerOpen(true);
             }}
-            currency={outputCurrency}
+            currency={outputCurrency()}
```

Comment:
instead of calling inputCurrency() and outputCurrency(), please call it at the top

Reply:
Not sure, I got this comment. Do you mean invoking it to top?

Reply:
yes correct @akkikumar72. it should be invoke only at the top once instead of everywhere

Reply:
To check if I got it correctly

`const outputCurrency = getOutputCurrency()`

 on top?